ecodse_funtime_alpha/data.py

```diff
@@ -3,23 +3,61 @@
 
```

Comment:
From improved readability do `from os import path`

Reply:
But I don't want to import path